packages/app-bridge/src/react/useCoverPage.ts

```diff
@@ -26,7 +26,7 @@ export const useCoverPage = (
 
     const fetchCoverPage = useCallback(async () => {
         setIsLoading(true);
-        setCoverPage(await appBridge.getCoverPage());
+        setCoverPage(await appBridge.getCoverPage(appBridge.context('currentLanguage').get()));
```

Comment:
Couldn’t we just call `getCoverPage()` and use the updated context value from within the appBridge function?

Reply:
By default it uses the context in the implementation, so it is not necessary here.

<img width="605" alt="Screenshot 2023-11-29 at 14 36 41" src="https://github.com/Frontify/brand-sdk/assets/6846950/4d17bfae-fb7b-4d7c-833f-e79e21f2230a">

Will remove it 👍